src/python/sdk/econia_sdk/utils/decimals.py

```diff
@@ -12,56 +13,152 @@ def _verify_integer_input(integer_input: int):
+    min_size_nominal: str,
+    base_decimals: int,
+    quote_decimals: int,
+) -> Tuple[int, int, int]:
     """
-    Given a decimal representation of the smallest possible unit and the
-    whole unit's decimals (e.g. ETH has 18 decimals), return the tick size.
+    Returns the tuple of market parameters in integer terms, rather than nominal
+    terms, given the three nominal parameters plus base and quote decimals. These
+    can be directly used to register a market (along with the base/quote types).
 
     Parameters:
-    * `smallest_decimal_unit`: Decimal (e.g. "0.001" for one thousandth) of
-      the smallest possible unit relative to a whole unit.
-    * `coin_decimals`: The number of decimals one whole unit of the coin
-      has (e.g. USDC has 6 decimals, ETH has 18).
+    * `lot_size_nominal`: The size granularity in nominal base terms.
+    * `tick_size_nominal`: The tick granularity in nominal quote terms.
+    * `min_size_nominal`: The minimum size in nominal base terms.
+    * `base_decimals`: The number of decimals in the nominal base unit.
+    * `quote_decimals`: The number of decimals in the nominal quote unit.
+
+    Returns: (lot_size_integer, tick_size_integer, min_size_integer)
     """
-    return get_lot_size_integer(smallest_decimal_unit, coin_decimals)
+    lot_size = _get_lot_size_integer(size_precision_nominal, base_decimals)
+    tick_size = _get_tick_size_integer(
+        size_precision_nominal,
+        price_precision_nominal,
+        quote_decimals
+    )
+    min_size = _get_min_size_integer(min_size_nominal, base_decimals, lot_size)
+    return (lot_size, tick_size, min_size)
 
 
-def get_min_size_integer(
-    smallest_decimal_size: str, base_coin_decimals: int, lot_size: int
-) -> int:
+def get_market_parameters_nominal(
+    lot_size_integer: int,
+    tick_size_integer: int,
+    min_size_integer: int,
+    base_decimals: int,
+    quote_decimals: int,
+) -> Tuple[Decimal, Decimal, Decimal]:
+    """
+    Returns the tuple of market parameters in nominal terms, rather than integer
+    terms, given the three parameters as integers plus base and quote decimals.
+
+    Parameters:
+    * `lot_size_integer`: The size granularity in terms of base subunits.
+    * `tick_size_integer`: The tick granularity in terms of quote subunits.
+    * `min_size_integer`: The minimum size in terms of lots of base.
+    * `base_decimals`: The number of decimals in the nominal base unit.
+    * `quote_decimals`: The number of decimals in the nominal quote unit.
+
+    Returns: (lot_size_nominal, tick_size_nominal, min_size_nominal)
+    """
+    _verify_integer_input(lot_size_integer)
+    _verify_integer_input(tick_size_integer)
+    _verify_integer_input(min_size_integer)
+    _verify_integer_input(base_decimals)
+    _verify_integer_input(quote_decimals)
+    lot_size_nominal = Decimal(lot_size_integer) / (10 ** base_decimals)
+    tick_size_nominal = Decimal(tick_size_integer) / (10 ** quote_decimals)
```

Comment:
This changed when I ran it through the formatter

How about following the checklist at `src/python/sdk/README.md`?